packages/web/src/components/stat-banner/StatBanner.tsx

```diff
@@ -199,6 +207,21 @@ export const StatBanner = (props: StatsBannerProps) => {
     </Button>
   )
 
+  const { data: user } = useGetUserById(
+    { id: profileId! },
+    { disabled: !profileId }
+  )
+
+  const handleComposeClicked = useComposeChat({
+    user: user as User,
+    onInboxUnavailable: (user) =>
+      openInboxUnavailableModal({
+        userId: user.user_id,
+        onSuccessAction: createChatModalActions.open({}),
```

Comment:
does this mean we're opening the create chat modal after the user closes the inbox unavailable modal? is that what we want?

Reply:
Oh that might have been bad copy paste. Ty

Reply:
nw. also don't we want to only open the inbox unavailable modal if `!canCreateChat` and call `onMessage` if they can actually chat?

Reply:
useComposeChat handles that for us, so we can either remove onMessage or just open the unavailable modal more directly

Reply:
oh nice ok

Reply:
found a better way. lmao.

one more look please 🙏